src/app/loginComponents/accounts/external/accounts.component.ts

```diff
@@ -238,4 +247,45 @@ export class AccountsExternalComponent implements OnInit, OnDestroy {
     this.ngUnsubscribe.next();
     this.ngUnsubscribe.complete();
   }
+
+  revokeToken() {
+    let dialogRef = this.dialog.open(RevokeTokenDialogComponent, {
+      width: '600px',
+      data: { revokeButtonClicked: this.revokeButtonClicked },
```

Comment:
I don't think you need the data at all; this is for passing input to the dialog, but the dialog doesn't need any input -- you're always going to start with the button unclicked.